### ISSUE TITLE
Prisma errors running tasks/run-e2e

### ISSUE THREAD

**Issue by Tobbe** [redwoodjs/redwood#2748]
Running the e2e task I'm seeing a couple of Prisma related errors

```
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\rw dev --fwd=--open=false
  × Generating the Prisma client...
    → info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Command failed with exit code 1: yarn prisma generate --schema="C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\api\db\schema.prisma"
Error:
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Object
    at new NodeError (node:internal/errors:363:5)
    at validateString (node:internal/validators:119:11)
    at Object.join (node:path:429:7)
    at getDotPrismaDir (C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.RedwoodJSFrameworkPackages\packages\create-redwood-app\template\node_modules\@prisma\cli\prisma-client\generator-build\index.js:74018:26)
    at Object.generateClient (C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.RedwoodJSFrameworkPackages\packages\create-redwood-app\template\node_modules\@prisma\cli\prisma-client\generator-build\index.js:74039:49)
    at Object.onGenerate (C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.RedwoodJSFrameworkPackages\packages\create-redwood-app\template\node_modules\@prisma\cli\prisma-client\generator-build\index.js:78643:28)
    at LineStream.<anonymous> (C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.RedwoodJSFrameworkPackages\packages\create-redwood-app\template\node_modules\@prisma\cli\prisma-client\generator-build\index.js:2912:40)
    at LineStream.emit (node:events:394:28)
    at addChunk (node:internal/streams/readable:312:12)
    at readableAddChunk (node:internal/streams/readable:287:9)


error Command failed with exit code 1.
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\prisma generate --schema=C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\api\db\schema.prisma
Prisma schema loaded from db\schema.prisma
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\cross-env NODE_ENV=development yarn dev-server
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\cross-env NODE_ENV=development webpack-dev-server --config ../node_modules/@redwoodjs/core/config/webpack.development.js --open=false
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\rw-gen-watch
$ C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.bin\dev-server
```

And a few of these

```
api | TypeError: debugLib is not a function
api |
api |
api | 1 anonymous
api |   C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\.prisma\client\index.js:21
api |
api | 2 Module._compile
api |   C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\pirates\lib\index.js:99
api |
api | 3 Object.newLoader [as .js]
api |   C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\pirates\lib\index.js:104
api |
api | 4 anonymous
api |   C:\Users\tobbe\AppData\Local\Temp\redwood-e2e\2021-06-06T14-42-43.347Z\node_modules\@prisma\client\index.js:1
api |
```

Since I'm on Windows and can't run the actual e2e tests I can't verify if/how these errors would affect the actual tests, but just putting it up here for future reference

**Comment by Tobbe** [comment]
Found a Prisma issue (https://github.com/prisma/prisma/issues/6188) that talked about esbuild, so I tried that as well, but got the same "debugLib" error.

**Comment by thedavidprice** [comment]
Not seeing this on Mac.

I'd say if you're able to run yarn rw dev this is most likely a false positive.

**Comment by thedavidprice** [comment]
This should be resolved via #2963

**Comment by janpio** [comment]
What are you supposed to do until this is merged? We are experiencing on our e2e tests for Redwood on Vercel (with Prisma), and do not really know how to fix it. Even in a newly created Redwood project this seems to happen. https://github.com/prisma/e2e-tests/pull/1911/files

**Comment by thedavidprice** [comment]
@janpio You _shouldn't_ be experiencing this issue as this is specific to Redwood's [run-e2e script](https://github.com/redwoodjs/redwood/blob/main/tasks/run-e2e) and has to do with the way we have to 1) create a test project using the branch template codebase and then 2) link the branch packages' code with the new test project.

We have made a significant number of changes to the Redwood Framework dependencies and package structure. We've also added a lot of new TS generation+support. Typically any problems these have created are resolved by clearing out installed node_modules and (when applicable) deleting and re-creating yarn.lock (for projects that are upgrading). 

Maybe it's a cache issue in your case?

Regardless, I'd be happy to connect real-time to discuss and diagnose. Will DM you now.
